scripts/md2html/md2html.js

```diff
@@ -306,9 +323,10 @@ for (let l in lines) {
         let heading = 0;
         while (line[heading] === '#') heading++;
         let delta = heading-prevHeading;
+        if (delta>1) console.warn(delta,line);
         if (delta>0) delta = 1;
         //if (delta<0) delta = -1;
-        if (Math.abs(delta)>1) console.warn(delta,line);
+        // if (Math.abs(delta)>1) console.warn(delta,line);
```

Comment:
Any particular reason to keep this line as a comment instead of delete?

Reply:
Reverence for @MikeRalphson 😄

Reply:
Will remove them in a future PR